ajax_select/static/ajax_select/js/ajax_select.js

```diff
@@ -24,7 +31,7 @@
 
       function addKiller(repr, pk) {
         var killId = 'kill_' + pk + id,
-            killButton = '<span class="ui-icon ui-icon-trash" id="' + killId + '">X</span> ';
+            killButton = killButtonTemplate.replace('{{ killId }}', killId);
```

Comment:
simpler would be:

`killButton = (options.killButtonTemplate || '<span class="ui-icon ui-icon-trash" id="{{killId}}">X</span> ').replace('{{killId}}', killId);`
 
no need for the delete options.killButton etc.